src/WormholeRouter.sol

```diff
@@ -139,17 +128,10 @@ contract WormholeRouter {
     function settle(bytes32 targetDomain, uint256 batchedDaiToFlush) external {
         bytes32 sourceDomain = domains[msg.sender];
         require(sourceDomain != bytes32(0), "WormholeRouter/sender-not-bridge");
```

Comment:
Let's lose the bridge terminology. Error can be `WormholeRouter/unsupported-source-domain`.